docs/sdk-and-tools/rest-api/transactions.md

```diff
@@ -479,3 +521,333 @@ Transaction status retrieved successfully.
 :::important
 The optional query parameter **`sender`** is only applicable to requests against the Proxy (not against the Observer Nodes).
 :::
+
+## <span class="badge badge-primary">GET</span> **Get Transactions Pool**
+
+`http://local-proxy-instance/transaction/pool`
+
+:::warning
+This endpoint isn't available on public gateway. However, it can be used on a local proxy instance, by setting `AllowEntireTxPoolFetch` to `true`
+:::
+
+This endpoint allows one to fetch the entire transactions pool, merging the pools from each shard.
+
+### Default
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Request-->
+
+Query Parameters
+
+| Param     | Required                                  | Type     | Description                          |
+|-----------|-------------------------------------------|----------|--------------------------------------|
+| fields    | <span class="text-normal">OPTIONAL</span> | `string` | A list of the fields to be included. |
+
+Example:
+
+`http://local-proxy-instance/transaction/pool`
+
+<!--Response-->
+
+🟢 200: OK
+
+Transaction status retrieved successfully.
+
+```
+{
+  "data": {
+    "txPool": {
+      "regularTransactions": [
+        {
+          "txFields": {
+            "hash": "84bb8a..."
+          }
+        },
+        {
+          "txFields": {
+            "hash": "4e2c43..."
+          }
+        }
+      ],
+      "smartContractResults": [],
+      "rewards": []
+    },
+    "error": "",
+    "code": "successful"
+}
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+### Using custom fields
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Request-->
+
+Query Parameters
+
+| Param     | Required                                  | Type     | Description                          |
+|-----------|-------------------------------------------|----------|--------------------------------------|
+| fields    | <span class="text-normal">OPTIONAL</span> | `string` | A list of the fields to be included. |
+
+s seen above, if the `fields` item is empty, only the transaction hash will be displayed.
+
+Example request with fields:
+
+`https://gateway.elrond.com/transaction/pool?fields=sender,receiver,value`
+
+All possible values for fields item are:
+
+- hash
```

Comment:
hash is already default, nothing happens if added to fields

Reply:
I know, but it's more clear this way